src/journal/journald-server.c

```diff
@@ -525,6 +529,7 @@ static void do_vacuum(Server *s, JournalStorage *storage, bool verbose) {
 
         (void) cache_space_refresh(s, storage);
 
+        FIXME: doesn't we want to show the disk usage after vacuuming ?
```

Comment:
Yeah, I think that'd be reasonable. That message is just one line, so it's not particularly obtrusive, and shouldn't happen too often, and the information is useful.